main.go

```diff
@@ -7,19 +7,25 @@ import (
 	"strconv"
 	"time"
 
+	"github.com/PrasadG193/covaccine-notifier/pkg/notify"
 	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
```

Comment:
```suggestion
	"github.com/pkg/errors"
	"github.com/spf13/cobra"
	
	"github.com/PrasadG193/covaccine-notifier/pkg/notify"
```
Please import internal packages in separate grp